src/main/java/domain/CrewGroup.java

```diff
@@ -0,0 +1,37 @@
+package domain;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class CrewGroup {
+    private List<Crew> crews;
+
+    private CrewGroup(List<Crew> crews) {
+        this.crews = crews;
+    }
+
+    public static CrewGroup from(List<String> crewNames) {
+        if (crewNames.stream().distinct().count() != crewNames.size()) {
+            throw new IllegalArgumentException("중복된 이름의 크루는 존재할 수 없습니다.");
+        }
+
+        List<Crew> crews = crewNames.stream().map(crewName -> new Crew(crewName)).toList();
+        return new CrewGroup(crews);
```

Comment:
변수 할당은 최소화 하는것이 좋아요!
이유는 GC(가비지 컬렉션) 에 대해서 학습해보면 알 수 있습니다 ㅎㅎ

- [JVM에 관하여 - Part 4, Garbage Collection 기초](https://tecoble.techcourse.co.kr/post/2021-08-30-jvm-gc/)

```suggestion
        return new CrewGroup(crewNames.stream().map(crewName -> new Crew(crewName)).toList());
```

Reply:
인텔리제이 자동완성을 사용하면서 놓친 부분이 있었네요........ 꼼꼼히 확인하고 GC 공부도 해볼게요 감사해요!